db/migrate/20160829175601_add_content_filter_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddContentFilterToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :content_filter, :string
```

Comment:
This should really be handled by a separate object where many of them can point to a given user.